supla-common/proto.h

```diff
@@ -680,6 +727,48 @@ typedef struct {
   unsigned _supla_int_t disablesLocalOperation;
 } TActionTriggerProperties;
 
+#define SUPLA_HVAC_VALUE_FLAG_SETPOINT_TEMP_MIN_SET (1ULL << 0)
+#define SUPLA_HVAC_VALUE_FLAG_SETPOINT_TEMP_MAX_SET (1ULL << 1)
+// Tells if output responsible for heating function is enabled
+#define SUPLA_HVAC_VALUE_FLAG_HEATING (1ULL << 2)
+// Tells if output responsible for cooling function is enabled
+#define SUPLA_HVAC_VALUE_FLAG_COOLING (1ULL << 3)
+// Tells if channel works in Weekly Schedule mode. If set to 0, then it works
+// in manual mode
+#define SUPLA_HVAC_VALUE_FLAG_WEEKLY_SCHEDULE (1ULL << 4)
+#define SUPLA_HVAC_VALUE_FLAG_COUNTDOWN_TIMER (1ULL << 5)
+// FAN flag tells if fan function is enabled
+#define SUPLA_HVAC_VALUE_FLAG_FAN_ENABLED (1ULL << 6)
+#define SUPLA_HVAC_VALUE_FLAG_ERROR       (1ULL << 7)
+#define SUPLA_HVAC_VALUE_FLAG_CLOCK_ERROR (1ULL << 8)
+
+// HVAC modes are used in channel value (as a command from server or
+// as a status response from device to server) and in weekly schedules
+// programs. Programs can't use value TURN_ON and WEEKLY_SCHEDULE
+#define SUPLA_HVAC_MODE_NOT_SET         0
+#define SUPLA_HVAC_MODE_OFF             1
+#define SUPLA_HVAC_MODE_HEAT            2
+#define SUPLA_HVAC_MODE_COOL            3
+#define SUPLA_HVAC_MODE_AUTO            4
+#define SUPLA_HVAC_MODE_FAN_ONLY        6
+#define SUPLA_HVAC_MODE_DRY             7
+// TURN ON mode is a command. Device will use it to turn on and then it will
+// restore previous mode which was used before turn off.
+#define SUPLA_HVAC_MODE_CMD_TURN_ON         8
+// Weekly schedule is a command. Device will use it to enable weekly schedule
+// mode and then it will set its mode according to schedule
+#define SUPLA_HVAC_MODE_CMD_WEEKLY_SCHEDULE 9
+
+typedef struct {
+  unsigned char IsOn;  // DS: 0/1 (or 0..100 ?)
+  unsigned char Mode;                     // SUPLA_HVAC_MODE_
+  _supla_int16_t
+      SetpointTemperatureMin; // * 0.01 Celcius degree - used for heating
+  _supla_int16_t
+      SetpointTemperatureMax;    // * 0.01 - Celcius degree used for cooling
+  unsigned _supla_int16_t Flags; // SUPLA_HVAC_VALUE_FLAG_
+} THVACValue;
```

Comment:
Add to the test to verify that it does not exceed the size specified by SUPLA_CHANNELVALUE_SIZE.
https://github.com/SUPLA/supla-core/blob/master/supla-common/test/ProtoTest.cpp#L250